sdk/core/azure-core/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Bugs Fixed
 
 - [[#4213]](https://github.com/Azure/azure-sdk-for-cpp/issues/4213) Fixed a bug where `Host` request header is not set for non-default port (80, 443).
+- [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows.
```

Comment:
This CL entry lacks specifics to a reader that doesn't know the context, and seems a bit generic to me. Can we add the following:
```suggestion
- [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows by avoiding scalability issues associated with `std::locale::classic`.
```

Reply:
I wouldn't put the locale info - this isn't saying much more to the user. When they use the SDK, they are mot much aware that they are using locale-related functionality. If they want details, they'll click the link.
However, the symptom they may see is that CPU usage is higher than before, if they upgraded earlier, or why they may want the update, for which potential benefit.